usefulUtilities/statusIndicator/webApp/app/statusIndicator_webApp.js

```diff
@@ -192,43 +211,35 @@ function getAllEmployees(response) {
             return response.end(JSON.stringify(responseObject));
         }
 
-        var teamName = "";
-        var responseHTML = "";
-
-        if (results.length === 0) {
-            responseHTML = `<h2>There are no employees!</h2>`;
-        }
+        var responseObject = {
+            "status": "success",
+            "teams": []
+        };
 
+        var currentTeamObject = {};
+        var currentTeamName = false;
         for (var i = 0; i < results.length; i++) {
-            if (teamName !== results[i].teamName) {
-                // different team name
+            if (currentTeamName !== results[i].teamName) {
```

Comment:
Making sure I understand what is happening here.   currentTeamName starts out always being false, then if there is a team name it gets replaced.  Is currentTeamName being false handled else where(the tbd?)

Reply:
right now, currentTeamName can't be false (i.e. there's always a value returned from the DB, even if it's "TBD"). so, there's no need to handle that case right now. however, you're totally right to point this out; i could definitely see a need in the future to handle that case.